frontend/src/components/AutoSuggestion.tsx

```diff
@@ -0,0 +1,75 @@
+import { autocomplete } from '@algolia/autocomplete-js'
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions'
+import '@algolia/autocomplete-theme-classic'
+import { debounce } from 'lodash'
+import React from 'react'
+import { useEffect, useRef } from 'react'
+import { client } from '../lib/algoliaClient'
+import './Autosuggestion.css'
+
+interface SearchProps {
+  // eslint-disable-next-line no-unused-vars
+  onChange: (query: string) => void
+  placeholder: string
+  initialValue?: string
+  indexName: string
+  onReady: () => void
+}
+
+const Autocomplete = React.memo(
+  ({ onChange, placeholder, initialValue = '', indexName, onReady }: SearchProps) => {
+    const containerRef = useRef<HTMLDivElement | null>(null)
+    const searchRef = useRef<ReturnType<typeof autocomplete> | null>(null)
+
+    const debouncedOnChange = debounce((query: string) => {
+      onChange(query)
+    }, 1000)
+
+    useEffect(() => {
+      return () => {
+        debouncedOnChange.cancel()
+      }
+    }, [debouncedOnChange])
+
+    useEffect(() => {
+      if (containerRef.current) {
+        const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+          searchClient: client,
+          indexName: indexName,
```

Comment:
For now, use the hardcoded value `'issue_suggestions'` instead of the` indexName`